contracts/MetaCoin.sol

```diff
@@ -1,44 +0,0 @@
-pragma solidity ^0.4.2;
-
-import "@gnosis.pm/gnosis-core-contracts/contracts/Utils/Math.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Events/EventFactory.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Tokens/EtherToken.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Oracles/CentralizedOracleFactory.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Oracles/UltimateOracleFactory.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/MarketMakers/LMSRMarketMaker.sol";
-import "@gnosis.pm/gnosis-core-contracts/contracts/Markets/StandardMarketFactory.sol";
```

Comment:
oh yeah you're right. hmm... we can leave it for now if you want. Will have to specify import in deploy contracts.